components/TilePlayer.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useRef, useState, useEffect } from 'react'
 import Image from 'next/image'
 import dynamic from 'next/dynamic'
+
 import {
   AudioPlayer,
   ImageContainer,
   Controls,
   SliderContainer,
   StyledSlider,
   VolumeIcon,
-} from './tilePlayer.style'
+  // @ts-ignore
+} from './TilePlayer.style'
```

Comment:
Due the file change name, Typescript was throwing an error (File name consistency) but I suspect it was a false alarm since there was no other with `tilePlayer`